python-package/lightgbm/compat.py

```diff
@@ -11,16 +11,12 @@
 except ImportError:
     PANDAS_INSTALLED = False
 
-    class pd_Series:
+    class pd_Series: # type: ignore
```

Comment:
```suggestion
    class pd_Series:  # type: ignore
```

please add two spaces before each of these comments, to match our linting rules

https://github.com/microsoft/LightGBM/pull/3916/checks?check_run_id=1894201789

> ./python-package/lightgbm/compat.py:14:21: E261 at least two spaces before inline comment
./python-package/lightgbm/compat.py:17:24: E261 at least two spaces before inline comment
./python-package/lightgbm/compat.py:48:24: E261 at least two spaces before inline comment
./python-package/lightgbm/compat.py:122:22: E261 at least two spaces before inline comment
./python-package/lightgbm/compat.py:125:26: E261 at least two spaces before inline comment
./python-package/lightgbm/compat.py:128:23: E261 at least two spaces before inline comment